app/models/activity.rb

```diff
@@ -0,0 +1,4 @@
+# Activity model for customisation & custom methods
+class Activity < PublicActivity::Activity
+  has_many :notifications, inverse_of: :activity, dependent: :destroy
```

Comment:
> activity is a kind of notification

does that sound correct? That implies that all activities are notifications. Please figure out the cardinality of the associations correctly. Associations are not just thrown into the application.